### PR TITLE
Improve Helpers

### PR DESCRIPTION
**Pull request by szepeviktor**
@terrylinooo Fix various errors, make the code shorter and more readable.

### CODE REVIEW

File: src/Firewall/Helpers.php [hunk 1]
```diff
@@ -783,7 +781,5 @@ function get_session_id(): string
  */
 function create_session_id(): string
 {
-    $hash =  rand() . 'ej;1zj47vu;3e;31g642941ek62au/41' . time();
-
-    return md5($hash);
-}
\ No newline at end of file
+    return md5(rand() . SESSION_ID_SALT . time());
```
Comment (jdreesen) [inline src/Firewall/Helpers.php:784]: Wouldn't it be better to use a CSPRNG here (like `bin2hex(random_bytes(25))`)?

**Comment by terrylinooo** [comment]
Thank you. I will reivew your contribution later.

File: src/Firewall/Helpers.php (hunk 1 above)
Comment (szepeviktor) [reply src/Firewall/Helpers.php:784]: Definitely **yes**.

This PR only fixes small and big problems.

**Comment by terrylinooo** [comment]
Unit testing failed. I will carefully check them one by one.

**Comment by szepeviktor** [comment]
I'm sorry, I thought Scrutinizer runs the unit tests.

**Comment by terrylinooo** [comment]
@szepeviktor It's okay. Thank you very much for contributing. I have just implemented GitHub Action, it may help to check the PRs. This PR is closed becasue too many changes. It is better to split them into serveral small PRs.
You can also do unit testing for the changes.
Here is the step by step guide:
https://github.com/terrylinooo/shieldon/wiki/Contributing

**Comment by szepeviktor** [comment]
Okay! You can still pick changes by hand.

**Pull request closed without merging**